src/index.js

```diff
@@ -15,9 +15,20 @@ const SIMMERJS = fs.readFileSync(require.resolve('simmerjs/dist/simmer.js')).toS
 let _config = null;
 
 function injectNWTL(browser) {
-    browser.execute(DOM_TESTING_LIBRARY_UMD);
+    const shouldInject = browser.execute(function () {
```

Comment:
Related to this issue

https://github.com/testing-library/nightwatch-testing-library/issues/315